_posts/2023-01-11-install-and-manage-applications.md

```diff
@@ -11,555 +11,245 @@ published: true
-
-```bash
-apx unexport --dnf --bin <package>
-```
-
-### Installing Applications in the Arch Linux Container
-
-- You can install applications in the Arch Linux container using the following command:-
-
-```bash
-apx install --aur <packages>
-```
-
-- This command will automatically detect the desktop file entry in the package and add it to the Application menu and Vanilla Control Center.
-
-- Alternatively, you can install the applications by entering the Arch Linux container using the following commands:-
-
-```bash
-apx enter --aur
-sudo pacman -Syu <packages>
-exit
-apx export --aur <packages>
-```
-
-- You can export the installed binary to use it with the host directly using the following command:-
-
-```bash
-apx export --aur --bin <package>
-```
-
-### Uninstalling Applications from the Arch Linux Container
-
-- You can uninstall applications from the Arch Linux container using the following command:-
-
-```bash
-apx remove --aur <package>
-```
-
-- This command will automatically detect and remove the desktop file entry.
-
-- Alternatively, you can uninstall the applications by entering the Arch container using the following commands:-
-
-```bash
-apx enter --aur
-sudo pacman -Rs <packages>
-exit
-apx unexport --aur <packages>
-```
-
-- You can unexport the exported binary from the host using the following command:-
-
-```bash
-apx unexport --aur --bin <package>
-```
+### Updating Applications
 
-### Installing Applications in the Alpine Linux Container
+Updates are visible in the `Updates` panel in GNOME Software. In Vanilla OS, Updates are available for Flatpaks, Snaps and System Packages through it.
 
-- You can install applications in the Alpine Linux container using the following command:-
+Automatic Updates are configured in the `Update Preferences` tab located in the burger menu in the top right corner.
 
-```bash
-apx install --apk <packages>
-```
+![GNOME Software Update Proferences](/assets/uploads/Applications_Post/gnome-software-update-preferences.webp)
 
-- This command will automatically detect the desktop file entry in the package and add it to the Application menu and Vanilla Control Center.
+### Removing Applications
 
-- Alternatively, you can install the applications by entering the Alpine Linux container using the following commands:-
+You can remove the installed Flatpaks and Snaps from the `Installed` panel.
 
-```bash
-apx enter --apk
-sudo apk add <packages>
-exit
-apx export --apk <packages>
-```
+(**_Note_**: Some native applications installed on the host cannot be removed using this method)
 
-- You can export the installed binary to use it with the host directly using the following command:-
+## Manage AppImages
 
-```bash
-apx export --apk --bin <package>
-```
+### Installing AppImages
 
-### Uninstalling Applications from the Alpine Linux Container
+You can download AppImages from project pages on GitHub. Once downloaded, they won't have to be installed since they are executable binaries.
```

Comment:
Maybe mentioning [appimagehub](https://www.appimagehub.com/) is useful here?

Reply:
I personally think appimagehub is inherently against what AppImage is trying to achieve, so I'm not sure whether it should be listed as an "official" source.

Reply:
I don't think it really is against the goals of AppImage, since it's just a new source for AppImages, not the main or centralized source for them.